pandas/core/ops.py

```diff
@@ -477,16 +477,16 @@ def _get_op_name(op, special):
 Examples
 --------
 >>> a = pd.DataFrame([1, 1, 1, np.nan], index=['a', 'b', 'c', 'd'],
-                     columns=['one'])
+...                  columns=['one'])
 >>> a
    one
 a  1.0
 b  1.0
 c  1.0
 d  NaN
 >>> b = pd.DataFrame(dict(one=[1, np.nan, 1, np.nan],
-                          two=[np.nan, 2, np.nan, 2]),
-                     index=['a', 'b', 'd', 'e'])
+...                       two=[np.nan, 2, np.nan, 2]),
+...                 index=['a', 'b', 'd', 'e'])
```

Comment:
May need one more space here, so that the "i" in "index" is below the "d" in "dict'.